examples/cybersecurity/bug_bounty_workflow.py

```
@@ -0,0 +1,59 @@
+"""
+Advanced Bug Bounty Workflow Example - CAI Framework
+---------------------------------------------------
+This script demonstrates a complete bug bounty workflow using CAI with
+HackerOne and Bugcrowd integration.
+"""
+
+from cai.types import Agent
```

Ruff (0.11.9)

`cai.types.Agent` imported but unused

Remove unused import: `cai.types.Agent`

(F401)